docs/development/security/gdpr_regulation.md

```diff
@@ -0,0 +1,156 @@
+* Another choice would be to establish an organisation with commercial intent, whether for-profit or non-profit, which would then provide the GIRAF system services, and thereby operate it as a Software-as-a-Service (SaaS). Those involved in such an organisation, could be anyone knowledgeable about the GIRAF system, such as a subset of developers. The reason for this to be required if GIRAF was to be provided as a SaaS, is two-fold; firstly, to operate a SaaS, an entity must take the responsibility as the data controller and processor, along with any professional obligations, such as dealing with expenses, thereby leading to the second point, namely that it is unlikely that it is allowed to use the current AAU ITS servers for commercial purpose, and/or delegating the responsibilities of GDPR regulation to that department.
+
+Based on these observations, it is most likely that the first choice is the most appropriate one to take, as this would allow the institution interested in the GIRAF system to assume control and responsibility of the systems and data, along with allowing the current structure of the GIRAF project to continue; as nearly everyone involved as developers, are only active for a four month period once a year, after which they are replaced with the next group of students studying that semester.
+
+The definitions are therefore as follows. The data controller would be the institution seeking to make use of the GIRAF project, by running the system on their own set of servers, thereby also letting the institution assume the role as data processor. This could, based on the specific institutions wishes be delegated to an external provider, thereby making that provider the data processor. The primary data subjects is the citizens of the particular institution, while secondary data subjects would be any caretakers registered in the system.
+
+The personal information processed in the system is data such as the name of the citizen (elaborate with any other information), along with the unique calendar assigned.
+
+### Essential Principles
+
+The GDPR  regulation states several essential principles, by which every data controller should ensure their business processes operate. Based on the previous discussion of definitions, the primary responsibilities falls upon the institutions adopting the GIRAF system, but in order for any organisation to be willing to do so, the GIRAF system must be designed and implemented with the regulation in mind, which is therefore the responsibility of the development team.
+
+The principles is summarized to the following list of tasks for the GIRAF project:
```

Comment:
```suggestion
The principles are summarised to the following list of tasks for the GIRAF project:
```

Reply:
Changed to s instead of z for consistency with you mostly writing in British English.